crates/wasi-common/src/sys/windows/path.rs

```diff
@@ -170,7 +170,7 @@ pub(crate) fn link(
     let new_path = concatenate(new_dirfd, new_path)?;
     if follow_symlinks {
         // in particular, this will return an error if the target path doesn't exist
-        log::debug!("Following symlinks for path: {:?}", old_path);
+        tracing::debug!("Following symlinks for path: {:?}", old_path);
```

Comment:
```suggestion
        tracing::debug!(
            old_path = tracing::field::display(&old_path),
            "Following symlinks"
        );
```
(I'm less inclined to suggest that `concatenate` should get the `old_path = ...` treatment because I wonder if that's actually useful logging?)